test/file05.tsx

```diff
@@ -0,0 +1,56 @@
+import { Story } from '@storybook/react';
+import { StepAnswers } from '@xsite-ui/common.questionnaire-manager';
+import { INPUT_TYPES } from '@xsite-ui/prequal-funnels.constants';
+import React, { useState } from 'react';
+
+import questionProps from '../../mocks/home-insurance-address.json';
+import Address from '../../src/address';
+import { QuestionProps } from '../../src/types';
+import { getPixelLabel } from '../../src/utils';
+import { useInvalidContext } from '../hooks/use-invalid-context';
+import { StyledCta } from '../styles';
+
+const HomeInsuranceAddress: Story<QuestionProps> = () => {
+  const { isInvalid, isInvalidTimeOut, isInvalidHandler } = useInvalidContext();
+  const { questions } = questionProps;
+  const { config, id = '' } = questions[0];
+  const { profileKey = '' } = config ?? {};
+
+  const [answer, setAnswer] = useState<StepAnswers>({});
+
+  const onBlurInput = (answer: StepAnswers) => {
```

Comment:
The onBlurInput function is redefining the 'answer' parameter that may cause confusion.